TKeazirian.HTTPServer/Server/Server.cs

```diff
@@ -6,13 +6,21 @@ namespace TKeazirian.HTTPServer.Server;
 
 using Request;
 
-public static class Server
+public class Server
 {
-    private static string? _clientRequest;
+    private string _clientRequest;
     public const string LocalIpAddress = "127.0.0.1";
     public const int Port = 5000;
+    private readonly Router _router;
 
-    public static void StartListening()
+    public Server()
+    {
+        _clientRequest = "";
```

Comment:
does the clientRequest really need to be a class variable?

Reply:
No I can refactor that a bit!